platform/src/main/java/org/stellar/anchor/platform/config/PropertyAssetsConfig.java

```diff
@@ -66,6 +71,18 @@ public void validate(@NotNull Object target, @NotNull Errors errors) {
               String.format("assets.type:%s is not supported.", this.getType()));
           break;
       }
+
+      assert assetService != null;
```

Comment:
Can we do `errors.reject()` instead of stopping the JVM?

Reply:
This line is to get away with the NullPointerException warning from the following line `assetService.listAllAssets()`.
assetService here will never be null actually, the prior switch case have captured all the null case.

Let me know if this makes sense to you :) Or we can remove this line